### PR TITLE
Anomalys drop cores again

### LINKED ISSUE #7714: Neutralized Anomalies No Longer Drop Cores
Anomaly cores no longer drop from neutralized anomalies, preventing ever building a phazon even with event RNG.

I looked and haven't seen anywhere a mention that this is intended, even the rnd overhaul mentions it's meant to rely on anomaly events now but they don't drop the cores.

### PR DESCRIPTION
**Pull request by SamHPurp**
Seems it was being garbage collected in error in #7038

Fixes #7714

:cl: Purpose2
fix: Anomalies will once again drop cores.
/ :cl:

### CODE REVIEW

File: code/game/objects/effects/anomalies.dm [hunk 1]
```diff
@@ -21,7 +21,6 @@
 	poi_list |= src
 
 /obj/effect/anomaly/Destroy()
-	QDEL_NULL(aSignal)
```
Comment (Crazylemon64) [inline code/game/objects/effects/anomalies.dm]: Could you please leave in
`aSignal = null`?

Comment (Fox-McCloud) [inline code/game/objects/effects/anomalies.dm]: Yeah, do this.

Comment (SamHPurp) [inline code/game/objects/effects/anomalies.dm]: Good point. Done.

**Comment by Fox-McCloud** [comment]
```
 /obj/effect/anomaly/proc/anomalyNeutralize()
 	new /obj/effect/effect/bad_smoke(loc)
 
 	for(var/atom/movable/O in src)
 		O.loc = src.loc
 
qdel(src)
```

😒 who wrote this.

**Pull request merged**